expr/LICENSE

```diff
@@ -0,0 +1,25 @@
+Regarding type expr, much of parse.go, parse_test.go:
+
+Copyright (c) 2014,2015 Damian Gryski <damian@gryski.com>
```

Comment:
add "Copyright (c) 2017 Grafana Labs"  to indicate that this is a derivative based on Damian's original code.